includes/admin/class-wc-stripe-command-palette-controller.php

```
@@ -0,0 +1,54 @@
+			$asset_metadata['dependencies'],
+			$asset_metadata['version'],
+			true
+		);
+		wp_set_script_translations(
+			'wc-stripe-command-palette',
+			'woocommerce-gateway-stripe'
+		);
+
+		wp_enqueue_script( 'wc-stripe-command-palette' );
+	}
+}
```

PHPMD (2.15.0)

[error] The class WC_Stripe_Command_Palette_Controller is not named in CamelCase. (undefined)

(CamelCaseClassName)